gem/tests/test_views.py

```diff
@@ -369,6 +378,37 @@ def test_comment_shows_user_display_name(self):
         self.assertContains(response, "this is my alias")
         self.assertNotContains(response, "tester")
 
+    def test_anonymous_comment_with_alias(self):
+        self.client.login(username='tester', password='tester')
+        self.user.profile.alias = 'this is my alias'
+        self.user.profile.save()
+        data = MoloCommentForm(self.user, {}).generate_security_data()
+        data.update({
+            'comment': 'Foo',
+            'submit_anonymously': '1',
+        })
+        self.client.post(
+            reverse('molo.commenting:molo-comments-post'), data)
+        [comment] = MoloComment.objects.filter(user=self.user)
+        print(comment)
+        self.assertEqual(comment.comment, 'Foo')
+        self.assertEqual(comment.user_name, 'Anonymous')
+
+    def test_anonymous_comment_translation(self):
```

Comment:
Since we are not testing for the poodle translation I don't think this test is necessarily